src/dbt/kipptaf/models/students/intermediate/int_students__graduation_path_codes.sql

```diff
@@ -244,22 +244,24 @@ with
             and s.discipline = n.discipline
     )
 
-select
+select distinct

```

Comment:
why distinct?

Reply:
I have 4 kids that are getting dupes and I cannot figure out why. I checked every single CTE and they only show dups on the last one. I thought I had maybe missed a union but nope. I then checked to see if I had missed an rn_highest, but nope. I couldn't figure out why it was dupping just shoe kids, and since it is only 4 kids and I didn't want to keep overriding Walters' work on coding kids on PS every day, I pushed it for update.

Reply:
yeah we gotta figure out the dupes first, hopefully I can get to it today

Reply:
Found the dup!